atlants_page_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rspec'
+require 'watir'
+require 'selenium-webdriver'
+
+browser = Watir::Browser.new :chrome
+
+RSpec.configure do |config|
+  config.color = true
+  config.formatter = :documentation
+  config.before(:each) { @browser = browser }
+  config.before(:all) {browser.window.maximize}
+  config.after(:suite) { browser.close unless browser.nil? }
+end
+
+describe 'Navigator regression test' do
+	context 'When opening Atlantbh web page' do
+
+		it 'should open Navigator' do
+			@browser.goto('http://www.navigator.ba/')
+		end
+
+		it 'should check if properly redirected' do
+			expect(@browser.url).to eq('http://www.navigator.ba/#/categories')
+			puts "Redirected to:" + @browser.url
+		end
+
+		it 'should open the page of Atlantbh' do
+			@browser.a(:xpath => '//*[@id="ember628"]/div[3]/div[4]/div/a[2]').click
+		end
+
+		it 'should check if the page is opened' do
+			@browser.window(:url => /atlantbh/).use

```

Comment:
In this step browser hangs. Seems like 'atlantbh' page is not being loaded for some reason. If I add a quick @browser.refresh it reloads the page and test is passing.

Reply:
It works for me without it, but I added it to the script.